Frontend/src/css/App.css

```diff
@@ -10,7 +10,6 @@ h3 {
 body {
   margin: 0;
   padding: 0;
-  overflow: hidden;
```

Comment:
Even if you want to remove this line for testing, I strongly encourage we use it on the final product.